rpc/types.go

```diff
@@ -325,10 +323,10 @@ type ContractClass struct {
 
 	EntryPointsByType EntryPointsByType `json:"entry_points_by_type"`
 
-	Abi *ContractABI `json:"abi,omitempty"`
+	Abi *ABI `json:"abi,omitempty"`
 }
 
-type ContractABIEntry interface {
+type ABIEntry interface {
```

Comment:
when we have type names that aren't ambiguous, i.e. dont exist in other contexts, we should prefer shorter type names